website/siteConfig.js

```diff
@@ -194,7 +194,11 @@ const siteConfig = {
   highlight: {
     theme: 'atom-one-dark',
   },
-  scripts: ['https://buttons.github.io/buttons.js'],
+  scripts: [
+    'https://buttons.github.io/buttons.js',
+    'https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.0/clipboard.min.js',
+    '/js/code-blocks-buttons.js',
+  ],
```

Comment:
You have to add the stylesheets here as well.

Reply:
fixed

Reply:
> docusaurus-init copies files from examples/basics, so the scripts and styles files have to be added to that folder too. 

spoke too soon, oh i see now...i will fix this and try a new docusaurus-init